coral/src/domain/team/team-api.ts

```diff
@@ -1,8 +1,36 @@
 import api, { API_PATHS } from "src/services/api";
-import { KlawApiResponse } from "types/utils";
+import { KlawApiRequest, KlawApiResponse } from "types/utils";
 
 const getTeams = () => {
   return api.get<KlawApiResponse<"getAllTeamsSU">>(API_PATHS.getAllTeamsSU);
 };
 
-export { getTeams };
+const updateTeam = ({
+  userName,
+  teamId,
+}: {
+  userName: string;
+  teamId: number;
+}) => {
+  const payload = {
+    username: userName,
+    teamId: teamId,
+  };
+  console.log("hello");
+  return api.post<
+    KlawApiResponse<"updateUserTeamFromSwitchTeams">,
+    KlawApiRequest<"updateUserTeamFromSwitchTeams">
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
```

Comment:
Why do we need those `ts-ignore` comment? :o

Reply:
The openapi definition claims that it needs the full `UserModel` but for this it only needs `username` and `teamId`. We don't have the full `UserModel` available in coral and since it contains passwords we decided against that. It would be a follow up ticket for the backend to have the right request body here later.

Reply:
I think a comment saying exactly that would be very useful, as it is very unusual to disable TS like this :o